go-controller/pkg/ovn/egressip_test.go

```diff
@@ -1956,14 +1966,16 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations cluster default network"
 							ExternalIDs: getEgressIPLRPNoReRoutePodToNodeDbIDs(IPFamilyValueV4, types.DefaultNetworkName, fakeOvn.controller.eIPC.controllerName).GetExternalIDs(),
 						},
 						&nbdb.LogicalRouter{
-							Name:  types.GWRouterPrefix + node1.Name,
-							UUID:  types.GWRouterPrefix + node1.Name + "-UUID",
-							Ports: []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1.Name + "-UUID"},
+							Name:    types.GWRouterPrefix + node1.Name,
+							UUID:    types.GWRouterPrefix + node1.Name + "-UUID",
+							Ports:   []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node1.Name + "-UUID"},
+							Options: logicalRouterOptions,
 						},
 						&nbdb.LogicalRouter{
-							Name:  types.GWRouterPrefix + node2.Name,
-							UUID:  types.GWRouterPrefix + node2.Name + "-UUID",
-							Ports: []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node2.Name + "-UUID"},
+							Name:    types.GWRouterPrefix + node2.Name,
+							UUID:    types.GWRouterPrefix + node2.Name + "-UUID",
+							Ports:   []string{types.GWRouterToJoinSwitchPrefix + types.GWRouterPrefix + node2.Name + "-UUID"},
+							Options: logicalRouterOptions,
```

Comment:
Do we expect `dynamic_neigh_routers` to have an impact on egress ip? Is that why you added the option here in egressip_test.go and not in the other unit tests we have?

Reply:
Not particularly. I think there are other egress IP tests that need to be patched to use the Options field too. But a grep across the repo didn't show me other files affected. I would like some feedback though on if hybrid overlay should also have this change?

Reply:
Yes, or for winc containers use-cases.

Reply:
Uhm. I wonder if this option in unit tests makes any difference :)  If it does, then all gateway routers in all unit tests in the repo should be updated, right? Either that or I'm missing a few pieces here. :)

Reply:
the other unit tests probably rely on generateGatewayInitExpectedNB which is in gateway_test.go. egress IP must be a one off.

Reply:
Not sure I understand the context around hybrid overlay. We don't create a router for that.